app/ui/browser/views/page/page.jsx

```diff
@@ -171,13 +194,12 @@ function addListenersToWebView(webview, page, pageIndex, dispatch, ipcRenderer)
       case 'show-bookmarks':
         console.warn('@TODO: ipc-message:show-bookmarks');
         break;
-      case 'scroll':
-        dispatch(setPageDetails({
-          pageIndex,
-          scrollX: e.args[0].x,
-          scrollY: e.args[0].y,
-        }));
+      case 'scroll': {
+        const { y: scrollY } = e.args[0];
+        const chromeMode = scrollY ? 'collapsed' : 'expanded';
```

Comment:
So if we're scrolled -any- amount, this will kick in? If you think we'll need to change it by any amount, we should add some constant so we can tweak later, e.g. `scrollY > SCROLL_THRESHOLD ? 'collapsed' : 'expanded'`